examples/gans/pix2pix_facades.py

```diff
@@ -34,7 +35,7 @@
 _URL = "https://people.eecs.berkeley.edu/~tinghuiz/projects/pix2pix/datasets/facades.tar.gz"
 
 PATH_TO_ZIP = tf.keras.utils.get_file("facades.tar.gz", origin=_URL, extract=True)
-PATH = os.path.join(os.path.dirname(PATH_TO_ZIP), "facades/")
+PATH = Path(PATH_TO_ZIP).parent.joinpath("facades/")
```

Comment:
please replace all `joinpath` with `/` operator (it works both on `str` and `Path` types). Also, remove the final `/` after facades